docker/build/Dockerfile-deploy

```diff
@@ -6,7 +6,7 @@ WORKDIR /root
 RUN wget --quiet https://dl.google.com/go/go${GOLANG_VERSION}.linux-amd64.tar.gz && tar -xvf go${GOLANG_VERSION}.linux-amd64.tar.gz && mv go /usr/local
 ENV GOROOT=/usr/local/go \
     GOPATH=$HOME/go \
-    GOPROXY=https://gocenter.io,https://goproxy.io,direct
+    GOPROXY=https://gocenter.io
```

Comment:
maybe I'm missing something, but is that really related to network partitioning testing ?

Reply:
This fix is required to allow the `create_and_deploy_recipe.sh` script to function properly.

Reply:
was the `create_and_deploy_recipe.sh` broken ?
I'm pretty confident that your changes here doesn't affect the  `create_and_deploy_recipe.sh` directly, so I'm concerned about making changes that aren't needed.

If "direct" is broken, then we really need to figure why.

Reply:
Multipath GOProxy is not supported with go version 1.12.

Reply:
As we are going to shift to go 1.14 shortly, we may want to consider reverting this.

Reply:
Reverted back to multipath goproxy.